.github/workflows/gradle.yml

```diff
@@ -298,11 +298,15 @@ jobs:
           github-token: ${{ needs.find_valid_pat.outputs.pat }}
   frontend-build:
     name: "Frontend: Build"
-    needs: start-frontend-runner
+    needs:
+      - start-frontend-runner
+      - find_valid_pat
     runs-on: ${{ needs.start-frontend-runner.outputs.label }}
     steps:
       - name: Checkout Airbyte
         uses: actions/checkout@v2
+        with:
```

Comment:
Curious, why add this?

Reply:
It's for Chromatic to be able to track the history:
> Chromatic supports the latest versions of the actions/checkout (i.e., versions 2 and 3). Both of them come with a caveat. They will only retrieve a single commit without any additional history. Chromatic needs the full Git history to keep track of changes in your repository.
https://www.chromatic.com/docs/github-actions#support-for-codeactionscheckoutv2code-and-above